Procfile.dev

```diff
@@ -1,4 +1,4 @@
 web: rails s
 hot: sh -c 'cd client && npm start'
 client: sh -c 'rm app/assets/javascripts/generated/* || true && cd client && npm run build:dev:client'
-
+server: sh -c 'cd client && npm run build:dev:server'
```

Comment:
Note, we have to rebuild the server webpack file with `wepback -w`. See definition [here](https://github.com/shakacode/react_on_rails-generator-results/blob/redux-to-redux-server-rendering-comparison-1_1_1/client/package.json#L32):

```javascript
"build:dev:server": "webpack -w --config webpack.server.rails.config.js",
```